src/Api.hs

```diff
@@ -1,15 +1,21 @@
-{-# LANGUAGE DataKinds     #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE TypeOperators #-}
```

Comment:
We don't align on pragmas anymore. The diff will only show a new language pragma, not highlighting every line that was changed just to align the imports.